reduce.json

```diff
@@ -83,7 +72,7 @@
         }
     },
     "returns": {
-        "description": "A data cube with the newly computed values. The number of dimensions is reduced by one, except if a target dimension is specified. The resolution and cardinality are the same as for the original data cube.",
+        "description": "A data cube with the newly computed values. The number of dimensions is reduced by one. The resolution and cardinality are the same as for the original data cube.",
```

Comment:
> The number of dimensions is reduced by one. The resolution and cardinality are the same as for the original data cube."

"It has one dimension less than the original data cube, but the remaining dimensions have their resolution and cardinality preserved."

Reply:
Sounds good, fixed.